.gitignore

```
@@ -12,3 +12,6 @@ dxil.dll
 
 # Generated by "examples/scene/scene.rs"
 assets/scenes/load_scene_example-new.scn.ron
+
+crates/bevy_asset/imported_assets
```

I ended up with 265 meta files when I ran the sprite example.

```suggestion
assets/**/*.meta
crates/bevy_asset/imported_assets
```

---

I suspect we'll want exceptions for examples built to use processing, but this seems like the right first step.